modules/clients/src/test/java/org/apache/ignite/common/CacheEventSecurityContextTest.java

```diff
@@ -198,9 +198,7 @@ public void testIgniteClient() throws Exception {
             checkEvents(cli, k -> cache.removeAsync(k).get(), true, EVT_CACHE_OBJECT_REMOVED);
 
             checkEvents(cli, k -> cache.remove(k, "val"), true, EVT_CACHE_OBJECT_REMOVED);
-
-            // TODO Add test case inside transaction after resolving IGNITE-14317.
-            checkEvents(k -> cache.removeAsync(k, "val").get(), true, EVT_CACHE_OBJECT_REMOVED);
+            checkEvents(cli, k -> cache.removeAsync(k, "val").get(), true, EVT_CACHE_OBJECT_REMOVED);
```

Comment:
We should unignore this test case for server/client nodes as well.

See https://github.com/apache/ignite/pull/11238/files#diff-fc4f36fd74e0c8bd8308fb49bd1c751b504001b6e8ea776244abb4fe1d1a8948R342

Reply:
Thanks, fixed